### DISCUSSION TITLE
TabletNum数量不一致，且没有办法找到差异进行修正

### DISCUSSION THREAD

**Discussion by Ustinov-lwp** [apache/doris#21570] [A - General / Q&A] [answered]
![image](https://github.com/apache/doris/assets/20982097/ec7133f9-3bda-4492-8fb8-2f1892f1e4bb)

这个图中的Tabletnum数量不一致，系统本身无法自动修正，此问题可能会导致查询数据不一致或者在批量查询时CPU占用比其他主机高，不知道哪位大神可以解答一下如何删除多出来的Tablet

**Comment by yagagagaga** [comment, marked as answer]
tablet 不一定要一致，不一致可能的原因是你在建表时，指定的 buckets 数不是 `be 数量` 的倍数。查询时某些机器资源占用率高可能是因为有数据倾斜。

**Reply by Ustinov-lwp** [reply]
非常感谢   但是确实这个的CPU占用率明显高于其他机器，或许我应该去调整表的buckets数量来进行尝试
